File: 7-kyu/playing-cards-draw-order--part-1/javascript/solution.js
const draw = (deck) => {
	let drawnCards = [];
  let i = 0;
  while (deck.length > 0) {
    const topCard = deck.shift();
    if (i % 2 === 0) {
      drawnCards.push(topCard);
    } else {
      deck.push(topCard);
    }
    i++;
  }
  return drawnCards;
};

File: 7-kyu/playing-cards-draw-order--part-1/python/solution.py
def draw(deck):
    drawn_cards = []
    i = 0
    while deck:
        top_card = deck.pop(0)
        if i % 2 == 0:
            drawn_cards.append(top_card)
        else:
            deck.append(top_card)
        i += 1
    return drawn_cards
